src/main/java/Parser.java

```diff
@@ -0,0 +1,161 @@
+/**
+ * Represents a parser that parses the different user input.
+ */
+public class Parser {
+
+
+    /**
+     * Parses the input to check for "bye".
+     *
+     * @param input Input of the user.
+     * @return True if there is "bye".
+     */
+    public static boolean isBye(String input) {
+        return input.equals(TaskElement.BYE.label);
+    }
+
+
+    /**
+     * Parses the input to check for "list".
+     *
+     * @param input Input of the user.
+     * @return True if there is "list".
+     */
+    public static boolean isList(String input) {
+        return input.equals(TaskElement.LIST.label);
+
+    }
+
+    /**
+     * Checks the input if there is command done present.
+     *
+     * @param input Input of the user.
+     * @return True if there is "done".
+     */
+    public static boolean isDone(String input) {
+        return input.split(" ")[0].equals(TaskElement.DONE.label);
+    }
+
+    /**
+     * Checks the input if there is command Todo present.
+     *
+     * @param input Input of the user.
+     * @return True if there is "Todo".
+     */
+    public static boolean isToDo(String input) {
+        return input.split(" ")[0].equals(TaskElement.TODO.label);
+    }
+
+    /**
+     * Checks the input if there is command deadline present.
+     *
+     * @param input Input of the user.
+     * @return True if there is "deadline".
+     */
+    public static boolean isDeadline(String input) {
+        return input.split(" ")[0].equals(TaskElement.DEADLINE.label);
+    }
+
+    /**
+     * Checks the input if there is command event present.
+     *
+     * @param input Input of the user.
+     * @return True if there is "event".
+     */
+    public static boolean isEvent(String input) {
+        return input.split(" ")[0].equals(TaskElement.EVENT.label);
+    }
+
+    /**
+     * Checks the input if there is command delete present.
+     *
+     * @param input Input of the user.
+     * @return True if there is "delete".
+     */
+    public static boolean isDelete(String input) {
+        return input.split(" ")[0].equals(TaskElement.DELETE.label);
+    }
+
+    /**
+     * Parses the remaining input to check for "done".
+     *
+     * @param input Input of the user.
+     * @return True if there is "done".
+     */
+    public static boolean parseDone(String input) {
+        return input.split(" ")[0].equals(TaskElement.DONE.label);
+    }
```

Comment:
Would it have been better to use a clearer name for a boolean such as isParseDone?